src/main/java/com/google/sps/app/servlets/NewTripServlet.java

```diff
@@ -44,15 +48,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             writeToDatastore(newTrip);
             final Gson gson = new Gson();
             response.setContentType("application/json;");
-            response.getWriter().println(gson.toJson(newTrip.tripID()));
+            //response.getWriter().println(gson.toJson(newTrip.tripID()));
+            response.getWriter().print(newTrip.tripID());
         } 
         else {
             System.out.println(request.getParameter(START_DATE_PARAM));
             System.out.println("Input information error: "+ error);
             response.getWriter().println("Input information error: "+ error);
         }
 
-        response.sendRedirect("https://summer22-sps-36.appspot.com/");
+        //response.sendRedirect("/trip/tripDetails.html");
```

Comment:
No need to fix now, but if added back, redirect to trip dashboard when trip successfully created

Reply:
just deleted the redirect in Servlet, but added redirect to dashboard.html for now in tripEvent.js